src/internal/transforms/transforms.go

```diff
@@ -43,12 +43,12 @@ func bijectiveMap(inputDir, outputDir string, pm PathMapper, dm DataMapper) erro
 		if err != nil {
 			return errors.EnsureStack(err)
 		}
-		defer inputFile.Close()
+		defer inputFile.Close() //nolint:errcheck
```

Comment:
Final comment on Close :D... this is why I wanted to migrate the linters to nogo because I felt like golangci-lint was suppressing errors by default that I don't think should be suppressed.  I hope that going forward, someone will write this code, it won't compile, and they'll think "oh yeah, that error could matter".